From d0c09dc4ca0376e415cf6a4cfcd3e26a136a9ba6 Mon Sep 17 00:00:00 2001
From: user1-github <56021366+user1-github@users.noreply.github.com>
Date: Wed, 3 Sep 2025 20:59:02 +0300
Subject: [PATCH] Patch makefile to make Tesseract find SDL2_image in /app/lib

---
 source/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/Makefile b/source/Makefile
index a2e23ab..137e609 100644
--- a/source/Makefile
+++ b/source/Makefile
@@ -57,7 +57,7 @@ CLIENT_LIBS+= -framework SDL2_mixer -framework CoreAudio -framework AudioToolbox
 CLIENT_LIBS+= -framework AudioUnit -framework OpenGL -framework Cocoa -lz -Lenet -lenet
 else
 CLIENT_INCLUDES= $(INCLUDES) -I/usr/X11R6/include `sdl2-config --cflags`
-CLIENT_LIBS= -Lenet -lenet -L/usr/X11R6/lib -lX11 `sdl2-config --libs` -lSDL2_image -lSDL2_mixer -lz -lGL
+CLIENT_LIBS= -L/app/lib -Lenet -lenet -L/usr/X11R6/lib -lX11 `sdl2-config --libs` -lSDL2_image -lSDL2_mixer -lz -lGL
 endif
 endif
 ifeq ($(PLATFORM),LINUX)
